### PR TITLE
Create abhinavkashyap061.json

### PR DESCRIPTION
**Pull request by abhinavkashyap061**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1340"></a>

### CODE REVIEW

File: public/data/abhinavkashyap061.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "url": "https://abhinavkashyap061.github.io/",
+        "icon": "website"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhinavkashyap061/",
+        "icon": "linkedin"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/abhinavkashyap061.json:28]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/abhinavkashyap061.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "icon": "website"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhinavkashyap061/",
+        "icon": "linkedin"
+      }
+    ]
+
+}
```
Comment (Panquesito7) [inline public/data/abhinavkashyap061.json:29]: The endline is still not fixed. I think it should be fixed.
```suggestion
    ]
}

```

Comment (emmalearnscode) [reply public/data/abhinavkashyap061.json:29]: @Panquesito7 we have a GitHub action that fixes it, so no need

**Review by Susmita-Dey — APPROVED** [review]
LGTM.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Readable, Maintainable

**Review by vijay-jaisankar — APPROVED** [review]
LGTM!

**Review by avinashupadhya99 — APPROVED** [review]

**Pull request merged**